man/SAPHanaSR_maintenance_examples.7

```diff
@@ -79,14 +81,17 @@ Note: Former versions of the Linux cluster used "migrate" instead of "move" and
 .PP
 \fB*\fR Perform an SAP HANA takeover by using SAP tools. 
 
-The procedures is described here for scale-out. It works for scale-up as well. 
+The procedure is described here for scale-out. It works for scale-up as well. 
+The procedure will stop the HANA primary. This might take a while. If you want
+to avoid waiting for the stopped primary, use the below procedure which suspends
+the primary.
 The status of HANA databases, system replication and Linux cluster has to be
 checked.
 The SAP HANA resources are set into maintenance, an sr_takeover is performed,
 the old primary is registered as new secondary.
-Therefore the correct secondary site name has to be used, see later example.
+Therefor the correct secondary site name has to be used, see later example.
```

Comment:
possible typo - do you mean 'therefor' or better 'therefore'? as used later on in the text